tools/data-handler/src/utils/clingo-facts.ts

```diff
@@ -0,0 +1,260 @@
+import { sep } from 'path';
+import { Card } from '../interfaces/project-interfaces.js';
+import {
+  CardType,
+  FieldType,
+  Link,
+  LinkType,
+  Workflow,
+} from '../interfaces/resource-interfaces.js';
+import { ClingoProgramBuilder } from './clingo-program-builder.js';
+import { ClingoFactBuilder } from './clingo-fact-builder.js';
+import { Project } from '../containers/project.js';
+
+export const createWorkflowFacts = (workflow: Workflow) => {
+  const builder = new ClingoProgramBuilder().addFact('workflow', workflow.name);
```

Comment:
Could the fact names be public string literals/constants?

Reply:
Done